automation/src/test/scala/org/broadinstitute/dsde/test/api/BillingProjectOwnerPermissionSpec.scala

```diff
@@ -0,0 +1,260 @@
+package org.broadinstitute.dsde.test.api
+
+import org.broadinstitute.dsde.test.util.AuthDomainMatcher
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.config.UserPool
+import org.broadinstitute.dsde.workbench.fixture.{BillingFixtures, GroupFixtures, WorkspaceFixtures}
+import org.broadinstitute.dsde.workbench.service.BillingProject.BillingProjectRole
+import org.broadinstitute.dsde.workbench.service.Orchestration.groups.GroupRole
+import org.broadinstitute.dsde.workbench.service.{Rawls, RestException, Sam}
+import org.scalatest.{FreeSpec, Matchers}
+
+import scala.util.Try
+
+class BillingProjectOwnerPermissionSpec extends FreeSpec with WorkspaceFixtures with BillingFixtures with GroupFixtures with Matchers {
+
+  "removing permissions from billing project owners for workspaces with auth domains" - {
+
+    "+ project owner, + group member, create workspace, - group member" in {
+
+      val owner = UserPool.chooseProjectOwner
```

Comment:
`owner` is always the user who's permissions we care about. Sometimes they start as a billing project owner or a member of the auth domain or both. As such, `owner` should be changed to a different name like `user`, `student` or something else that indicates that this is the user who's permissions we are testing. On top of that, this user should always be chosen using `UserPool.chooseStudent`